src/frontend/src/lib/services/address.services.ts

```diff
@@ -69,22 +72,43 @@ const loadTokenAddress = async <T extends Address>({
 	return { success: true };
 };
 
+// We use the Testenet address for Regtest.
+type BtcTokenIdsWithAddresses = typeof BTC_MAINNET_TOKEN_ID | typeof BTC_TESTNET_TOKEN_ID;
+type BtcTokenIdsStringWithAddresses = Exclude<BitcoinNetwork, 'regtest'>;
```

Comment:
why `-WithAddresses` suffix in the two types above? in the sense that this type is "to be used with addresses"?

BTW, shouldn't the second one be something like `BtcNetworkIdsWithAddresses`?

Reply:
I wanted to indicate that those are the btc network that have a different address. Regtest doesn't have it's unique address. Naming is hard...

Yes, second makes more sense `BtcNetworksWithAddresses`, done!

Reply:
Ahahahah agreed! Naming IS hard!